pom.xml

```diff
@@ -2767,6 +2767,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>xmlunit</groupId>
+            <artifactId>xmlunit</artifactId>
+            <version>1.6</version>
```

Comment:
XMLUnit appears to have a 2.x version line (under the `org.xmlunit` group). You may want to use the 2.x line instead, since the 1.x line might not be maintained anymore. If you need the 1.x line's features, you could import the `xmlunit-legacy` JAR.

Reply:
OK, switched to 2.9.1